apps/web/package.json

```diff
@@ -6,17 +6,26 @@
     "dev": "next dev",
     "build": "next build",
     "start": "next start",
-    "lint": "next lint"
+    "lint": "eslint --fix './'",
+    "clean": "rm -rf .turbo && rm -rf node_modules"
   },
   "dependencies": {
-    "next": "^12.1.0",
+    "@babel/core": "^7.17.7",
+    "@emotion/react": "^11.8.2",
+    "@emotion/styled": "^11.8.1",
+    "@mui/material": "^5.5.1",
+    "next": "12.1.0",
+    "next-transpile-modules": "^9.0.0",
     "react": "^17.0.2",
```

Comment:
Why are these here? Aren't they meant to be peer dependencies of mui? How does that work?

Reply:
They **_are_** peer dependencies in `MUI` and `ui` and hence why they're here. Exactly, how does it [work](https://github.com/npm/cli/blob/latest/docs/content/configuring-npm/package-json.md#devdependencies)?